src/patternfly/components/Select/examples/index.js

```diff
@@ -83,6 +86,13 @@ export default props => {
       <Example heading="Select - single (expanded)" handlebars={SelectSingleExpandedExampleRaw} minHeight="20em">
         {selectSingleExpandedExample}
       </Example>
+      <Example
+        className="is-expanded-top"
+        heading="Select - single (top expanded)"
+        handlebars={SelectSingleTopExpandedExampleRaw}
```

Comment:
need to add `import SelectSingleTopExampleDoc from '../docs/select-single-top.md';` toward the top of this file, then `docs={SelectSingleTopExampleDoc}` here in this `<Example>` to hook up the documentation.

Reply:
Thanks!